src/main/java/com/acrolinx/sidebar/jfx/AcrolinxSidebarJFX.java

```diff
@@ -39,10 +39,11 @@
  */
 public class AcrolinxSidebarJFX implements AcrolinxSidebar
 {
+    protected static final Logger logger = LoggerFactory.getLogger(AcrolinxSidebarJFX.class);
```

Comment:
Any reason why this is not private?

Reply:
Not sure, maybe originally there was a subclass extending this class which needed access. Or the author made the common misconception that "protected" is a very restrictive and locked down visibility modifier in Java (which it is not, but I have seen many developers confusing this due to the name which implies a lot of "protection").